City-of-Helsinki/drupal-hdbt
# UHF-3117: Fix errand-service__detail-link icons on mobile

# What was the problem

[UHF-3117](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3117)

On mobile sizes, icons were too narrow on [some pages](https://helfi-kymp.docker.so/fi/kaupunkiymparisto-ja-liikenne/pysakointi/asukaspysakointi).

![image](https://user-images.githubusercontent.com/1191667/140503781-15b257e4-34f9-4680-96df-52f38562b7f9.png)


# What was done

Since icons were set to be flex elements, `flex-shrink: 0;` helps here.

# How to test

* Get kymp database
* Test [page with problem](https://helfi-kymp.docker.so/fi/kaupunkiymparisto-ja-liikenne/pysakointi/asukaspysakointi).
* Notice problem on narrow viewport (see image above)
* Get branch
* Refresh page, notice problem being soved
* Approve
